### PR TITLE
Cult can only use runes on station and on mining

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
I'm not sure this is a good idea but the best way to get attention for a bad idea is to open a PR

Cult runes can only be used on the station z level or the mining z level.

Why?

Because it's effectively impossible for security to fight cultists on the white ship, or who have their base in some other random bit of space debris. Hiding in an unassailable base on another z level for 30-40 minutes is bad.

> but then why'd you let them keep building on mining

Because I realize that there are not many good places to build a base in maint (hey maybe making multiple game modes about  base building in ss13 wasn't a great idea). Mining however

-Can support security running about on it. You don't need suits for lavaland.

-Is thematically appropriate for the veil being weak

-Is possible to actually navigate unlike 6 z levels of emptiness and random transitions.

-Leaves a trail when you dig out into it. So they can hide a base there, but not forever.

Organizing a raid to go shoot cultists in hell sounds more fun to me than "well I guess we should call the shuttle because they're in their mobile fortress we can't attack again"

Anyway tell me why it's a bad idea please so I can close this PR instead of being responsible for tanking the cult winrate.

:cl: Kor
rscadd: Cult runes can only be scribed on station and on mining.
/:cl:

### CODE REVIEW

**Comment by ChangelingRain** [comment]
Should allow use on centcom, too.

**Comment by Zilenan91** [comment]
It's good :+1: 

White ship cult is horrible in every way I can think of.

**Comment by demonfiren** [comment]
It's a bad PR because Zilenan thinks it's good :^)

**Comment by KorPhaeron** [comment]
Fair point, guess I gotta kill it now

**Comment by Zilenan91** [comment]
mr. gorbachev tear down this pull request

**Comment by ChangelingRain** [comment]
I'd like to note that clock cult, while it does gain a lot from using the white ship as a base, doesn't have the innate capacity to set up an instant teleportation network, as all the clockcult teleportation requires resources in some form.

**Comment by hornygranny** [comment]
:+1: we are basically back to originalcult at this point with derelictcult research teams

**Comment by Zilenan91** [comment]
Aren't the clock walls indestructible like cult ones are? Either way it's still a problem since they can sit there and not interact with the round until they have everything they could ever need to win. That's not the worst part either since if either cult has constructs they can just have a dude sending the shuttle over and over again to permastun everyone invading it while the constructs kick them while they're down.

**Comment by ChangelingRain** [comment]
clock cult walls are not indestructible, no

**Comment by Durkel** [comment]
What ever kills the white ship meme at this point will make me happy.

**Comment by InsaneHyena** [comment]
Yeah, I'm really not sure. On one hand, white ship memes are annoying. On the other, there are only so much places you can hide your cult shit, and validhunting assistants/sec will check them. Always. And higher the pop, the sooner somebody will do it.

**Comment by TechnoAlchemisto** [comment]
but kor what if I land the whiteship on lavaland and draw the runes on it there

**Comment by KorPhaeron** [comment]
You can't make runes on shuttles at all

**Comment by TechnoAlchemisto** [comment]
fucking fascists

**Comment by SidonCC** [comment]
Bend over backwards because the game's nature doesn't support modes like this

**Comment by Cheridan** [comment]
i don't really like this it feels like a bandaid fix but i also can't think of any way to fix it short of giving sec Space F-22s to intercept the white ship

**Comment by Kromgar** [comment]
See the issue is once cult is found all of maint is swarmed by sec as they find the base.

That is the main fuckign issue.

**Comment by KorPhaeron** [comment]
Well if cult is unplayable without avoiding the game, we'd have to admit to ourselves that it was a bad idea and finally remove it, but I know we won't do that, so we're gonna try this first

**Comment by ChangelingRain** [comment]
The solution is clearly adding more areas to the station in maintenance.

**Comment by KorPhaeron** [comment]
@optimumtact 

Okay I tested it

File: code/game/gamemodes/cult/ritual.dm [hunk 1]
```diff
@@ -172,9 +172,17 @@ This file contains the arcane tome files.
 	var/entered_rune_name
 	var/list/possible_runes = list()
 	var/list/shields = list()
+	var/area/A = get_area(src)
+
 	if(locate(/obj/effect/rune) in Turf)
 		user << "<span class='cult'>There is already a rune here.</span>"
 		return
+
+	if(Turf.z != ZLEVEL_STATION && Turf.z != ZLEVEL_MINING)
```
Comment (ChangelingRain) [inline code/game/gamemodes/cult/ritual.dm]: You should allow runes on centcom/the escape shuttle because that's more fun.

Comment (KorPhaeron) [inline code/game/gamemodes/cult/ritual.dm]: But I had to blacklist runes on shuttles

Comment (Zilenan91) [inline code/game/gamemodes/cult/ritual.dm]: otherwise that's teleport runes on the cargo shuttle to get ERT armory gear every round

**Comment by optimumtact** [comment]
Ruining the cult meta one pull at a time

**Pull request merged**